x-pack/plugins/security_solution/cypress/tasks/rule_details.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Exception } from '../objects/exception';
+import { RULE_STATUS } from '../screens/create_new_rule';
+import {
+  ADD_EXCEPTIONS_BTN,
+  CLOSE_ALERTS_CHECKBOX,
+  CONFIRM_BTN,
+  FIELD_INPUT,
+  FIELD_INPUT_RESULT,
+  LOADING_SPINNER,
+  OPERATOR_INPUT,
+  VALUES_INPUT,
+} from '../screens/exceptions';
+import {
+  ALERTS_TAB,
+  EXCEPTIONS_TAB,
+  REFRESH_BUTTON,
+  REMOVE_EXCEPTION_BTN,
+  RULE_SWITCH,
+} from '../screens/rule_details';
+
+export const activatesRule = () => {
+  cy.get(RULE_SWITCH).should('be.visible');
+  cy.get(RULE_SWITCH).click();
+};
+
+export const deactivatesRule = () => {
+  cy.get(RULE_SWITCH).should('be.visible');
+  cy.get(RULE_SWITCH).click();
+};
+
+export const addsException = (exception: Exception) => {
+  cy.get(LOADING_SPINNER).should('exist');
+  cy.get(LOADING_SPINNER).should('not.exist');
+  cy.get(FIELD_INPUT).should('exist');
+  cy.get(FIELD_INPUT).type(`${exception.field}{enter}`);
+  cy.get(FIELD_INPUT_RESULT).first().click();
```

Comment:
This was causing the tests to be flaky because sometimes it was not able to find a `first` element right away, which made the tests get stuck in the modal. I might be better to remove this line because I think it might not be needed.

Reply:
I'll take a look at this, thanks 😊